Remove "indicators query" from test modules (#15158)
demisto/content: Packs/EDL/ReleaseNotes/2_1_4.md

#### Integrations
##### Palo Alto Networks PAN-OS EDL Service

> - Fixed an issue where **test-modules** was failing when "Indicators Query" parameter wasn't supplied.

Reviewer: **test-module** - not plural.
In any case, I think it'll be more understandable if you say "where clicking on the **Test** button was ..."

> - Fixed an issue where clicking on the Test button was failing when "Indicators Query" parameter wasn't supplied.

Reviewer 2: ```suggestion
- Fixed an issue where the Test button would fail when the **Indicators Query** parameter wasn't supplied.
```
